src/ResourceManager/Network/Commands.Network/AzureRM.Network.psd1

```diff
@@ -389,7 +389,13 @@ CmdletsToExport = 'Add-AzureRmApplicationGatewayAuthenticationCertificate',
                'Set-AzureRmNetworkInterfaceTapConfig', 
                'Remove-AzureRmNetworkInterfaceTapConfig', 
                'Get-AzureRmVirtualNetworkTap', 'New-AzureRmVirtualNetworkTap', 
-               'Remove-AzureRmVirtualNetworkTap', 'Set-AzureRmVirtualNetworkTap'
+               'Remove-AzureRmVirtualNetworkTap', 'Set-AzureRmVirtualNetworkTap',
```

Comment:
you need to add the `az` versions of the cmdlets to the [`Az.Network.psd1`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network/Az.Network.psd1) file as well